tox.ini

```diff
@@ -14,7 +14,8 @@ commands = flake8 user_util
 [testenv]
 setenv = 
     PYTHONPATH = {toxinidir}
-deps = 
+deps =
+    setuptools 
```

Comment:
Rather than doing this here, I think it should be an updated in the relevant `requirements/*.in` files.

We should definitely add it to the `tox.in` file which is what's causing the package build to fail.

Reply:
Okay, I'll add and check it accordingly

Reply:
Also, it looks like part of the issue might be the really old version of tox.  It may be worth running `make upgrade` manually in this repo to get the dependencies up to dot.  It may be worth doing that in a separate PR.  That might fix it without requiring setuptools manually in `tox.in`

Reply:
After removing it from here and adding it in tox.in, if I check the run in nektos act, it still gives the same error

Reply:
@feanil  after running `make upgrade` other dependencies are not updated and adding `setuptools` back to this file the tests are now passing in the act. If I don't keep it in this file it still fails

I took reference from [here](https://github.com/openedx/edx-django-utils/pull/384/files#diff-ef2cef9f88b4fe09ca3082140e67f5ad34fb65fb6e228f119d3812261ae51449R73)